frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -212,12 +212,12 @@ export const funnelLogic = kea<funnelLogicType>({
                         ])
                         breakpoint()
                         insightLogic.actions.endQuery(queryId, ViewType.FUNNELS, result.last_refresh)
-
                         return { results: result.result, timeConversionResults }
                     } catch (e) {
                         if (!isBreakpoint(e)) {
                             insightLogic.actions.endQuery(queryId, ViewType.FUNNELS, null, e)
                         }
+                        console.error(e)
```

Comment:
debug log?

Reply:
This was actually intended as to not swallow the errors